### ISSUE TITLE
Observable of actions that completes interferes with other observables of actions

### ISSUE THREAD

**Issue by karptonite** [ngrx/store#337]
tl:dr: if you pass your store into the `subscribe()` method of an Observable that emits Actions, then **completes**, it improperly stops actions from other Observables of Actions that were similarly added to the store.

See this example:
https://plnkr.co/edit/ijTa6E7AY6z64MbjhqQ6?p=preview
The important code is in main.ts.

Clicks on "add person" emit events that are mapped to an action, then `store` subscribes to that stream of events, as an `Observer`.

Clicking "Kill store" creates an Observable that emits one action (that does nothing itself), then completes; store subscribes to this Observable. After you click it, clicks on "Add person" will no longer work.

By contrast, the "do not kill store" button does almost the same thing as the "kill store" button, except that its Observable does NOT complete.

Note that this bug only affects subscriptions if BOTH are added by passing store to the `subscribe()` method of an Observable of Actions. If you call `store.dispatch()` directly, it doesn't seem to be an issue.

**Comment by robwormald** [comment]
Please check this against NgRx v4, and if it’s still an issue, please reopen on https://github.com/ngrx/platform. Thanks!
